dataland-frontend/src/assets/scss/components/form.scss

```diff
@@ -504,7 +504,7 @@ input[type="checkbox"], input[type="radio"]{
     .formkit-options {
       display: flex;
       padding: .5rem 0 0 0;
-      .formkit-option:nth-child(2n) {
+      .formkit-option:nth-child(n+2) {
         margin-left: 10%;
```

Comment:
![image](https://user-images.githubusercontent.com/47553592/236821337-607f8687-818a-4b27-94ef-0e2575c2d698.png)
Can we make this to look more like an error?

Reply:
Yes I will start working on a solution

Reply:
Created a simple solution. Please check if it is ok like that for now.

Reply:
can we make that the "rejected" report doesn't get added to the "upload company reports" section in the first place? The Error message can stay as is, but the user should be able to remove it. Maybe also a modal could work.

Reply:
Created a simple solution based on a popup.
Tell me if it is ok for a first implementation.

This will break the tests now because they still expect the old stuff and I haven't adjusted them yet.
Since I'm pretty tired now, I'll do this after dinner and a break.

Reply:
Yes - the current implementation works for me.
Things left to do: 
* A TODO in the Code
* Design of the popup
* fixing all the tests

Reply:
You can check the feature now again.  It should be working stable and the styling of the popup should be ok.

Additionally, the UploadReports component is now a lot leaner and more maintainable.
Btw, I think it was a great idea to put some more time into this and not rush this on Monday night. Definitely paid off!

Reply:
ok